src/pages/user/authority/AuthorityOrcidModal.tsx

```diff
@@ -1,29 +1,25 @@
 import React, { FC, useState, useEffect } from 'react';
 import { useSelector } from 'react-redux';
 
-import { Authority } from '../../../types/authority.types';
 import AuthorityModal from './AuthorityModal';
 import OrcidModal from './OrcidModal';
 import { RootStore } from '../../../redux/reducers/rootReducer';
 
-interface AuthorityOrcidModalProps {
-  authority: Authority | null;
-}
-
 enum ModalType {
   AUTHORITY = 'authority',
   NONE = '',
   ORCID = 'orcid',
 }
 
-const AuthorityOrcidModal: FC<AuthorityOrcidModalProps> = ({ authority }) => {
+const AuthorityOrcidModal: FC = () => {
   const [openModal, setOpenModal] = useState(ModalType.AUTHORITY);
   const user = useSelector((store: RootStore) => store.user);
 
   useEffect(() => {
     if (user.authority) {
       // Set previouslyLoggedIn in localStorage to avoid opening this modal on every login
       localStorage.setItem('previouslyLoggedIn', 'true');
+      setOpenModal(ModalType.NONE);
```

Comment:
Det går vel ikke an å gjøre dette uansett, for da vil orcid-modalen aldri åpnes heller...

Reply:
kanskje best å legge en sjekk i App.tsx da, åpne OrcidModal hvis user.authority, ellers åpne AuthorityOrcidModal